.github/workflows/build-all.yml

```
@@ -36,12 +36,12 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - 
+        name: Generate containers from templates
+        run: ./generate-all.bash
```

appropriately named one-line run command makes name meaningless (same for the cases below)

```suggestion
      - run: ./generate-all.bash
```